conf/chunkserver.conf

```diff
@@ -107,8 +107,8 @@ copyset.scan_rpc_retry_times=3
 copyset.scan_rpc_retry_interval_us=100000
 # enable O_DSYNC when open chunkfile
 copyset.enable_odsync_when_open_chunkfile=false
-# sync timer timeout interval
-copyset.synctimer_interval_ms=30000
+# sync chunk limit default = 4 * 1024 * 1024
+copyset.sync_chunk_limits = 4194304
```

Comment:
remove the spaces around "="

Reply:
> remove the spaces around "="

Ok, Are my changes valid?

Reply:
Yes, the parameter "copyset.sync_chunk_limits" cloud be updated by curveadm now.

Reply:
> Yes, the parameter "copyset.sync_chunk_limits" cloud be updated by curveadm now.

thank you for your help.